.vscode/launch.json

```diff
@@ -1,17 +0,0 @@
-{
```

Comment:
Shouldnt we keep debugging?

Reply:
We should but i want to use my own launch .config and if every dev would do for differrent compilers / os the file will get very long. Maybe we should provide one or two examples in the technical documentation or the README.md instead.

Reply:
So let's add it to the .gitignore also right?